docker-compose/development/docker-compose.yml

```diff
@@ -94,3 +99,35 @@ services:
       - master-api
       - excel-export
     stdin_open: true
+
+  minio1:
+    image: minio/minio
+    volumes:
+      - ../../minio/data:/data
+    expose:
+      - "9000"
+    networks: 
+      - "trubudgetnetwork"
+    environment:
+      MINIO_ACCESS_KEY: minio
+      MINIO_SECRET_KEY: minio123
+    command: server /data
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:9000/minio/health/live"]
+      interval: 30s
+      timeout: 20s
+      retries: 3
+
+  nginx:
```

Comment:
@galethil After creating workflow item an error occurred:

URL: `http://localhost:3000/api/subproject.createWorkflowitem`

Method: POST

log from API:
```
minioClient.putObject Error: getaddrinfo ENOTFOUND nginx
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:66:26) {
  errno: 'ENOTFOUND',
  code: 'ENOTFOUND',
  syscall: 'getaddrinfo',
  hostname: 'nginx'
}
```

Note: "slim" is OK